torch/csrc/inductor/aoti_torch/utils.h

```diff
@@ -48,6 +48,17 @@ inline AtenTensorHandle new_tensor_handle(at::Tensor&& tensor) {
   return tensor_pointer_to_tensor_handle(new_tensor);
 }
 
+inline void assert_inf_and_nan(at::Tensor& check_tensor) {
+  auto flattened = check_tensor.view({-1});
+
+  for (int64_t i = 0; i < flattened.numel(); i++) {
+    auto value = flattened[i].item<float>();
+    if (std::isinf(value) || std::isnan(value)) {
+      assert(false);
```

Comment:
nit - can we throw an exception? ```assert``` may become a no-op if we compile the code in non-debug mode.

Reply:
changed to throw.